R/build-home.R

```diff
@@ -269,6 +270,29 @@ on_cran <- function(pkg, cran = cran_mirror()) {
   pkg %in% rownames(pkgs)
 }
 
+on_bioc <- function(pkg) {
```

Comment:
This is now quite duplicative with `on_cran()`. I'd recommend creating new functions called `in_repo()` and `bioc_mirror()` and eliminating both `on_cran()` and `on_bioc()`.